app/models/benchpress_stat.rb

```diff
@@ -1,4 +1,4 @@
 class BenchpressStat < Stat
-
+	belongs_to :stat
 
```

Comment:
This is a great idea, but not quite what we want here. Since BenchpressStat inherits from Stat, it essentially is already a stat so there's no need to set up relationships for them. Let me know if this all makes sense.